src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/http/AccessManagement.scala

```diff
@@ -8,21 +8,21 @@ object AccessManagement {
 
   private def getRequest(url: String): HttpRequestBuilder =
     http("/returnResourceAccessors")
-      .get("/api" + url)
+      .get("/api/v1" + url)
```

Comment:
Please define a global "version" variable and replace all of the version hardcoding in this file